src/jvm/main/org/jetbrains/kotlinx/lincheck/strategy/managed/modelchecking/ModelCheckingStrategy.kt

```diff
@@ -434,16 +434,21 @@ internal class ModelCheckingStrategy(
         }
     }
 
-    private inner class Choice(val node: InterleavingTreeNode, val value: Int)
+    private inner class Choice(val node: InterleavingTreeNode, val value: Int) {
+        override fun toString(): String {
+            return "Choice(node=$node, value=$value)"
+        }
+    }
 
     /**
      * This class specifies an interleaving that is re-producible.
      */
     private inner class Interleaving(
-        private val switchPositions: List<Int>,
-        private val threadSwitchChoices: List<Int>,
-        private var lastNotInitializedNode: SwitchChoosingNode?
+         private val switchPositions: List<Int>,
+         private val threadSwitchChoices: List<Int>,
+         private val initialLastNotInitializedNode: SwitchChoosingNode?
```

Comment:
Can it be only a constructor parameter, not a field?

Reply:
No, it cannot. See usages of this field.

Reply:
Let's add documentation to all these paramterers, please